prod/bsc/feeds.yaml

```diff
@@ -9860,27 +9860,27 @@ YEARN-FI:*-ETH:
 
 # options
 
-OP:ETH-*:
-  discrepancy: 1
-  precision: 18
-  inputs:
-    - fetcher:
-        name: OptionsPrice
-      calculator:
-        name: OptionsPrice
-        params:
-          sym: ETH
-
-OP:BTC-*:
-  discrepancy: 1
-  precision: 18
-  inputs:
-    - fetcher:
-        name: OptionsPrice
-      calculator:
-        name: OptionsPrice
-        params:
-          sym: BTC
+# OP:ETH-*:
+#   discrepancy: 1
+#   precision: 18
+#   inputs:
+#     - fetcher:
+#         name: OptionsPrice
+#       calculator:
+#         name: OptionsPrice
+#         params:
+#           sym: ETH
+
+# OP:BTC-*:
+#   discrepancy: 1
+#   precision: 18
+#   inputs:
+#     - fetcher:
+#         name: OptionsPrice
+#       calculator:
+#         name: OptionsPrice
+#         params:
+#           sym: BTC
```

Comment:
why not remove? we have git history if we want ot get back

Reply:
No particular reason, besides enabling it again after all external validators update. I'll delete it